src/components/SingleDatePicker.jsx

```diff
@@ -604,7 +559,7 @@ class SingleDatePicker extends BaseClass {
         )}
       >
         {enableOutsideClick && (
-          <OutsideClickHandler onOutsideClick={this.onClearFocus}>
+          <OutsideClickHandler onOutsideClick={this.onOutsideClick}>
```

Comment:
also here?

Reply:
Yup! I changed the naming of the `onClearFocus` method to `onOutsideClick` so it would match the nomenclature of the `DateRangePicker` component, but the method itself does the same thing.